src/core/Form/Select/BaseSelect/SelectItemList/SelectItemList.tsx

```diff
@@ -104,12 +104,12 @@ class BaseSelectItemList extends Component<
         role="listbox"
         onBlur={onBlur}
       >
-        <HtmlDivWithRef
-          forwardedRef={this.wrapperRef}
+        <HtmlSpanWithRef
```

Comment:
This should be outside of ul, if is possible to make it that way!

Reply:
Did you already tried if it does still work if doing that way?

Reply:
![image](https://user-images.githubusercontent.com/54316341/147938413-d2c9f3fa-524a-40d6-81f6-81312f0549cb.png)

Unfortunately does not seem to work with the span outside the list.

Reply:
As earlier discussed, the actual fix will be done in separate PR.